boards/FSM/Src/Io/Io_SteeringAngleSensor.c

```diff
@@ -0,0 +1,17 @@
+#include "Io_Adc.h"
+#include "configs/App_SteeringAngleConstants.h"
+#include "Io_SteeringAngleSensor.h"
+
+float Io_SteeringAngleSensor_GetAngleDeg(void)
+{
+    // Subtract the measured voltage by the steering angle sensor voltage
+    // offset. Multiply the result with VOLTAGE_TO_DEGREES to calculate the
+    // steering angle in degrees.
+    //
+    // The scale factor used to convert the voltage to an angle is given by 360
```

Comment:
Where did this scale factor come from?

Reply:
The scale factor is referring to `VOLTAGE_TO_DEGREES`. Should I be more specific in the javadoc

Reply:
I think you should just write it as an equation to avoid any misleading comments. Scale factor isn't really a standard terminology and isn't as specific as could be. The current comments is really just describing the formula but in a more verbose way.


```        
// Steering Angle =  (ADC Voltage - Voltage Offset) * Degree Per Volt

const float VOLTAGE_OFFSET = 1.9f;
const float DEGREE_PER_VOLT = 360.9 / 3.3f;

return  (Io_Adc_GetChannel1Voltage() - VOLTAGE_OFFSET ) *  DEGREE_PER_VOLT ;
```

Reply:
`DEGREE_PER_VOLT` seems a bit more clear than `VOLTAGE_TO_DEGREES` on a second though

Reply:
Ok should be fixed now